communication/slack-guidelines.md

```diff
@@ -111,6 +110,32 @@ and describe the situation.
 
 ---
 
+## Should you have a channel on the Kubernetes Slack?
+
+The primary purpose of the Kubernetes slack is for the coordination of the
+Kubernentes project. However it is useful for developers and users to have a
+strong ecosystem of channels for related things. Here are some guidelines for
+determining if you should request a channel:
+
+- The channel MUST be Kubernetes related in some way.
+  - Related cloud native projects might be more appropriate on the [CNCF Slack].
+- The project MUST be open source. 
+  - If you are open sourcing a project DO THAT FIRST, we cannot accomodate every
+    organization's open sourcing launch plans.
+  - The purpose of Slack is to organize an existing community, not seed new ones.
+  - Moderators look at contributor activity, adoption and community concensus
+    around a project, otherwise Slack becomes a vehicle for promotion instead of
+    promulgation.
+  - Establishing and maintaining a Slack channel should be an inflection point
+    in a project's adoption.
+  - Requesting a channel means maintaining it on behalf of the project filing
+    the issue, you will be expected to participate and foster a healthy
+    discourse.
+- Channels around commercial services built on OSS projects are allowed.
+  - Users love the value of being able to collaborate around various services.
+  - Keep it classy, on the Kubernetes Slack we're all on the same team.
+- Ask in #slack-admins or file an issue if you're unsure, it never hurts to ask.
+
```

Comment:
I would add:

```
- The channel MUST be public.
  - In order to ensure all channels adhere to our Code of Conduct, we heavily restrict private channels.
  - We do allow `#...-dev` channels for established projects where a single project channel is too noisy, but please don't create both at the start.
  - If you need private discussion areas for security-sensitive topics, a project-specific Slack or the [CNCF Slack] may be a better fit.
```

Reply:
+1 to those suggestions. 👍 

Otherwise lgtm

Reply:
Thank you so much for not just reviewing but giving me something copy and pasteable, <3.